src/Compilers/CSharp/Portable/Symbols/Source/SourceAssemblySymbol.cs

```diff
@@ -721,7 +725,9 @@ private void ValidateIVTPublicKeys(DiagnosticBag diagnostics)
                     {
                         if (oneKey.Key.IsDefaultOrEmpty)
                         {
+#nullable disable // Can 'oneKey.Value' be null? https://github.com/dotnet/roslyn/issues/39166
```

Comment:
The check should be `Key.IsEmpty`. We add a value on line 2129 in this file when the Key is empty.

Reply:
I'm planning to leave this for a follow-up